modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Models/PublisherExtensions.cs

```diff
@@ -127,7 +127,7 @@ public static class PublisherExtensions {
                 DataSetWriterGroup = endpoint.DataSetWriterGroup,
                 DataSetDescription = endpoint.DataSetDescription,
                 DataSetKeyFrameCount = endpoint.DataSetKeyFrameCount,
-                DataSetMetaDataSendInterval = endpoint.DataSetMetaDataSendInterval,
+                DataSetMetaDataSendInterval = endpoint.MetaDataUpdateTime,
```

Comment:
I'm not sure if we're missing the `MetaDataQueueName` here?

Reply:
Totally correct.  Will fix.